docs/guide/core-concepts.rst

```diff
@@ -980,6 +980,147 @@ that call ``geotrellis.vector`` home:
 -  ``geotrellis.vector.reproject`` defines methods for translating
    between projections
 
+Catalogs
+========
+
+We call the output of any ingest a *catalog*. It is a collection of
```

Comment:
The output of an ingest is a layer. I think when we ingest multiple layers into the same location using the same attribute store we start to call this collection a catalog, but mostly in the vernacular rather than in code.

This is conscious choice because we found that the idea of "catalog" is fairly nebulous from the standpoint of interface despite being intuitive. So we stuck to making API about layers, making them, moving them, reading them, deleting them.

We use the word "catalog" in configs and output directors fairly often but it is important to make this distinction for the sake of understanding the API